README.md

```diff
@@ -4,97 +4,87 @@
 [![License GPL-3.0](https://img.shields.io/badge/license-GPLv3.0-blue.svg)](https://github.com/MiPushFramework/MiPushFramework/blob/master/LICENSE)
 ![Min Android Version](https://img.shields.io/badge/android-lollipop-%23860597.svg)
 
-在非 MIUI 系统上体验小米系统级推送。
+Let supported push service run system-ly on every Android device
+Simplified Chinese document (简体中文文档): [README](https://github.com/MiPushFramework/MiPushFramework/blob/master/README_zh-rCN.md)
 
 ![](https://raw.githubusercontent.com/MiPushFramework/MiPushFramework/master/art/tab_events.jpg)
 ![](https://raw.githubusercontent.com/MiPushFramework/MiPushFramework/master/art/tab_permissions.jpg)
 ![](https://raw.githubusercontent.com/MiPushFramework/MiPushFramework/master/art/ask.jpg)
 ![](https://raw.githubusercontent.com/MiPushFramework/MiPushFramework/master/art/tab_settings.jpg)
 ![](https://raw.githubusercontent.com/MiPushFramework/MiPushFramework/master/art/tab_apps.jpg)
 
-## 什么是小米系统级推送，为什么会有这个项目
+## What is the Xiaomi system push service and the purpose of this project
 
-小米推送是小米公司提供的推送服务，许多 App 都在使用（如酷安）。
+Xiaomi push service is a Push service (like GCM) provided by Xiaomi, and there are lots of Chinese apps using it.
 
-它非常轻量，会在 MIUI 设备上自动启用系统推送，而非 MIUI 设备则在后台保持长连接。
+It is really lightweight, and it will start the system push service on MIUI ROMs, but apps will start a background service on non-MIUI ROMs.
```

Comment:
and it will start only one single background push service on MIUI firmwares instead of starting lots of background service for each user apps on non-MIUI phones

Reply:
英文好像没说清要表达什么。